### PR TITLE
fix(readme): rewrite readme.md for commit workflow(BFL-58)

### PR DESCRIPTION
**Pull request by weterbogov**
resolve #60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# BFL
+### BFL
+## Commit message rules
+All commits should have next template:
+`feature_or_fix(scope_of_task): summary of done work(name_of_project_with_number_of_task)`
+
+Some examples: 
+
+`feature(menu): add pop-up in-game menu functionality(BFL-52)`,
```
Comment (Verpad) [inline README.md]: If you write BFL-52 - it doesn't link commit to the task, so mb better to write just #52 or BFL #52

Comment (Verpad) [reply README.md]: @weterbogov yes it does
image

Comment (weterbogov) [reply README.md]: @Verpad I said about `commit` message. When you write in comment it is not actually `commit`

Comment (Verpad) [reply README.md]: @weterbogov it works wherever you write it
image

Comment (weterbogov) [reply README.md]: @Verpad it is not work in pull requests, but ok. I change rules for commits

Comment (Verpad) [reply README.md]: @weterbogov anyway pull request mostly used for closing tasks, not for referencing them

**Pull request merged**